extensions/amp-3q-player/OWNERS.yaml

```diff
@@ -0,0 +1,2 @@
+- cvializ
```

Comment:
let's put just alan for all video players

Reply:
is there anyone else we can put in? would like an alternative reviewer if possible

Reply:
done.